static/js/app.js

```diff
@@ -50,11 +92,43 @@ function addEventListeners() {
 
 	document.querySelectorAll(".delete-email-form").forEach(deleteForm => {
 		deleteForm.addEventListener("submit", deleteAliasConfirmation);
-	})
+	});
 }
 
+// Watch for the addon to update the dataset of <firefox-private-relay-addon></firefox-private-relay-addon>
+function watchForInstalledAddon() {
+	const installIndicator = document.querySelector("firefox-private-relay-addon");
+	const observerConfig = {
+		attributes: true
+	};
+
+	const patrollerDuties = (mutations, mutationPatroller) => {
+		for (let mutation of mutations) {
+			if (mutation.type === "attributes" && isRelayAddonInstalled()) {
+				hideAddToFxButtons();
+				showSecondarySignInButtons();
+				if (localStorage && !localStorage.getItem("addonInstalled", "true")) {
+					localStorage.setItem("addonInstalled", "true");
```

Comment:
What happens when the user uninstalls the add-on? How/when will we `localStorage.setItem("addonInstalled", false);`?

Reply:
I did a spot-check. This needs to be `sessionStorage`. I uninstalled the add-on and still saw "Sign in" buttons until I used developer tools to manually delete the localStorage.

Reply:
I did a spot-check. This needs to be sessionStorage. I uninstalled the add-on and still saw "Sign in" buttons until I used developer tools to manually delete the localStorage.